public/views/index/body.ejs

```diff
@@ -158,7 +158,7 @@
                         <%- __('Powered by %s', '<a href="https://codimd.org">CodiMD</a>') %> | <a href="<%- serverURL %>/s/release-notes" target="_blank" rel="noopener"><%= __('Releases') %></a> | <a href="<%- sourceURL %>" target="_blank" rel="noopener"><%= __('Source Code') %></a><% if(imprint) { %> | <a href="<%- serverURL %>/s/imprint" target="_blank" rel="noopener"><%= __('Imprint') %></a><% } %><% if(privacyStatement) { %> | <a href="<%- serverURL %>/s/privacy" target="_blank" rel="noopener"><%= __('Privacy') %></a><% } %><% if(termsOfUse) { %> | <a href="<%- serverURL %>/s/terms-of-use" target="_blank" rel="noopener"><%= __('Terms of Use') %></a><% } %>
                     </p>
                     <h6 class="social-foot">
-                        <%- __('Follow us on %s and %s.', '<a href="https://github.com/codimd/server" target="_blank" rel="noopener"><i class="fa fa-github"></i> GitHub</a>, <a href="https://community.codimd.org" target="_blank" rel="noopener"><i class="fa fa-users" aria-hidden="true"></i> Discourse</a>, <a href="https://riot.im/app/#/room/#codimd:matrix.org" target="_blank" rel="noopener"><i class="fa fa-comments"></i> Riot</a>, <a href="https://social.codimd.org/mastodon" target="_blank" rel="noopener"><i class="fa fa-mastodon"></i> Mastodon</a>', '<a href="https://translate.codimd.org" target="_blank" rel="noopener"><i class="fa fa-globe"></i> POEditor</a>') %>
+                        <%- __('Follow us on %s and %s.', '<a href="https://github.com/codimd/server" target="_blank" rel="noopener"><i class="fa fa-github"></i> GitHub</a>, <a href="https://community.codimd.org" target="_blank" rel="noopener"><i class="fa fa-users" aria-hidden="true"></i> Discourse</a>, <a href="https://app.element.io/#/room/#hedgedoc:matrix.org" target="_blank" rel="noopener"><i class="fa fa-comments"></i> Riot</a>, <a href="https://social.codimd.org/mastodon" target="_blank" rel="noopener"><i class="fa fa-mastodon"></i> Mastodon</a>', '<a href="https://translate.codimd.org" target="_blank" rel="noopener"><i class="fa fa-globe"></i> POEditor</a>') %>
```

Comment:
We should also replace "Riot" with "Matrix" or "Element"

Reply:
Good point. As matrix is the ecosystem and element just a client, I prefer replacing it with "matrix". Any other opinions?

Reply:
One thing that would speak for element: everyone who knows the details about Matrix will know what element is and how to convert an element link to something they can use with their clients. Someone who does _not_ know what matrix is has a pretty shitty first encounter with it (my opinion). So I'd prefer Element over Matrix as it is easier to get started with.

Reply:
As far as I understand the Link would be to element.io, but the discussion is about what the text should be. And there Matrix is a fine choice in my opinion.

Reply:
The alternative would be to include both names like "Element (matrix)".

Reply:
I think we could actually write "Matrix" and use the [matrix.to link](https://matrix.to/#/#hedgedoc:matrix.org?via=matrix.org&via=kif.rocks&via=chat.weho.st), as that got redesigned recently and is now way more usable than before.

Reply:
After seeing the redesign, yes please. We should use the matrix.to link.